templates/cluster-template-ipv6.yaml

```diff
@@ -0,0 +1,280 @@
+apiVersion: cluster.x-k8s.io/v1alpha3
+kind: Cluster
+metadata:
+  labels:
+    cni: ${CLUSTER_NAME}-crs-0
+  name: ${CLUSTER_NAME}
+  namespace: default
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks:
+      - 2001:1234:5678:9a40::/58
```

Comment:
As @aramase pointed out, this is with in the vnet range. It is documented in Calico that this is a requirement.  Should we call this out in the readme? https://docs.projectcalico.org/reference/public-cloud/azure

>Ensure that the selected pod’s subnet is a part of your Azure virtual network IP range.